dart/dynamics/FreeJoint.cpp

```diff
@@ -508,25 +508,9 @@ void FreeJoint::setAngularAcceleration(
 
 //==============================================================================
 Eigen::Matrix6d FreeJoint::getLocalJacobianStatic(
-    const Eigen::Vector6d& _positions) const
+    const Eigen::Vector6d& /*positions*/) const
```

Comment:
I think if ``positions`` isn't going to be used at all here, then we should either

1. Remove the ``positions`` argument entirely or
2.  Save the current positions in a temporary ``Eigen::Vector6d`` set the current positions to ``positions`` and then reset the positions to their original values at the end. If the incoming ``positions`` vector matches the current positions, then this shouldn't be very much overhead, because the bookkeeping should all shortcircuit.